src/qt/transactiondesc.cpp

```diff
@@ -57,25 +55,9 @@ QString TransactionDesc::FormatTxStatus(const CWalletTx& wtx)
             }
         }
 
-        if (wtx.IsLockedByLLMQInstantSend()) {
-            strTxStatus += " (" + tr("verified via LLMQ based InstantSend") + ")";
-            return strTxStatus;
-        }
-
-        if(!instantsend.HasTxLockRequest(wtx.GetHash())) return strTxStatus; // regular tx
-
-        int nSignatures = instantsend.GetTransactionLockSignatures(wtx.GetHash());
-        int nSignaturesMax = CTxLockRequest(wtx).GetMaxSignatures();
-        // InstantSend
-        strTxStatus += " (";
-        if(instantsend.IsLockedInstantSendTransaction(wtx.GetHash())) {
-            strTxStatus += tr("verified via InstantSend");
-        } else if(!instantsend.IsTxLockCandidateTimedOut(wtx.GetHash())) {
-            strTxStatus += tr("InstantSend verification in progress - %1 of %2 signatures").arg(nSignatures).arg(nSignaturesMax);
-        } else {
-            strTxStatus += tr("InstantSend verification failed");
+        if (wtx.IsLockedByInstantSend()) {
+            strTxStatus += " (" + tr("verified via InstantSend") + ")";
```

Comment:
Is this intentional? Won't this cause confusion for people who are used to see `LLMQ based` here? (also, should we remove `LLMQ based` for ChainLocks too in some future PR to keep things consistent?)

Reply:
It was intentional, but yeah, I guess you're right that this could cause confusion. I'll revert that part.

Reply:
I'm not sure tacking on LLMQ to these features is good from a usability perspective. LLMQ looks crazy, InstantSend anyone can understand.

Reply:
At least for the next release, community members might get confused when they see it changing back to just "InstantSend". I communicated a few times that we might decide to disable LLMQ based InstantSend if it turns out to not be good enough, so this is how such a change could be interpreted. But it should be fine to change it in the release following the next one.